From 319375ab22a555a6c739af9ca18d6087b7d79e29 Mon Sep 17 00:00:00 2001
From: Michael Niedermayer <michael@niedermayer.cc>
Date: Fri, 29 Sep 2017 02:28:43 +0200
Subject: [PATCH 5/7] avcodec/v4l2_context: Reduce spelling variations

Reviewed-by: Lou Logan <lou@lrcd.com>
Signed-off-by: Michael Niedermayer <michael@niedermayer.cc>
---
 libavcodec/v4l2_context.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libavcodec/v4l2_context.c b/libavcodec/v4l2_context.c
index d675c55f2b..297792f871 100644
--- a/libavcodec/v4l2_context.c
+++ b/libavcodec/v4l2_context.c
@@ -112,7 +112,7 @@ static inline void v4l2_save_to_context(V4L2Context* ctx, struct v4l2_format_upd
 }
 
 /**
- * returns 1 if reinit was succesful, negative if it failed
+ * returns 1 if reinit was successful, negative if it failed
  * returns 0 if reinit was not executed
  */
 static int v4l2_handle_event(V4L2Context *ctx)
@@ -266,7 +266,7 @@ static V4L2Buffer* v4l2_dequeue_v4l2buf(V4L2Context *ctx, int timeout)
             return NULL;
         }
         if (ret) {
-            /* if re-init was successfull drop the buffer (if there was one)
+            /* if re-init was successful drop the buffer (if there was one)
              * since we had to reconfigure capture (unmap all buffers)
              */
             return NULL;
-- 
2.14.2

